addons/html_editor/static/src/utils/dom_info.js

```diff
@@ -241,6 +241,18 @@ export function isSelfClosingElement(node) {
     return node && selfClosingElementTags.includes(node.nodeName);
 }
 
+/**
+ * Returns whether the given node is a element that could be visible when with
+ * empty text content.
+ *
+ * @param {Node} node
+ * @returns {boolean}
+ */
+const visibleEmptyTextElmentTags = ["BUTTON"];
+export function isVisibleEmptyTextElement(node) {
+    return node && visibleEmptyTextElmentTags.includes(node.nodeName);
+}
```

Comment:
The issue with un util like this one (a very generic name with a very specific implementation in practice) is that it should probably return `true` for other nodes as well, for example `img`, `i` with font awesome classes etc but it doesn't so it's kind of weird... ^^' Personally I would just add a call to the existing `isButton` util in `isEmptyBlock`, just like it already tests for `isMediaElement`.

Reply:
Ahh yes, you are right, I didn't want to make it generalized to `<a class="btn"></a>`, but they are basically the same thing and I just tested that we also have the empty button shown in this case. I will adapt it, thankssss :pray: